frontend/packages/schema-editor/src/components/SchemaInspector/EnumField.tsx

```diff
@@ -60,12 +59,14 @@ export const EnumField = ({
         error={!isValid}
       />
       {onDelete && (
-        <IconButton
-          ariaLabel={t('schema_editor.delete_field')}
+        <Button
+          title={t('schema_editor.delete_field')}
```

Comment:
I suggest keeping the aria-label attribute as it allows users who use screen readers to understand what the icon represents.

```suggestion
          title={t('schema_editor.delete_field')}
          aria-label={t('schema_editor.delete_field')}
```